server/UserRegistration.java

```diff
@@ -0,0 +1,467 @@
+package newbank.server;
+
+import com.sun.tools.javac.Main;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.*;
+import java.sql.SQLException;   // If have an error thrown from execution.
+
+public class UserRegistration extends Main {
+
+
+    public static UserRegistration newUser() throws SQLException {
+
+        Scanner sc = new Scanner(System.in);
```

Comment:
Since you use the `Scanner` in a lot of places, it would be better to make it a field of the `UserRegistration` class, and since it's only going to be used here, it can be marked as `private`

Reply:
You could for example create it inside of a constructor